packages/mux-player/src/template.ts

```diff
@@ -105,6 +105,7 @@ export const content = (props: MuxTemplateProps) => html`
           : false
       }"
       cast-stream-type="${isLive(props) ? 'live' : false}"
+      exportparts="video"
```

Comment:
is the reason why it's only needed here because it's not in the shadow dom of mux-player?

Reply:
yup, CSS parts need to be forwarded between multiple shadom doms

Reply:
I wish we didn't have to add such big attributes for themes but yea 🤷‍♂️